file_roots/debian_pkg.sls

```diff
@@ -3,6 +3,22 @@
 include:
 {% if buildcfg.build_release == 'debian8' %}
 
+{% if buildcfg.build_arch == 'armhf' %}
```

Comment:
Why the pkg list is different for armhf build_arch ?
Shouldn't it be the same as other architectures?

Does it mean the support of armhf devices is only partial at the moment?